### ISSUE TITLE
The first demo popup unable to open

### ISSUE THREAD

**Issue by YoungHF** [ghybs/Leaflet.MarkerCluster.LayerSupport#6]
maxzoom  : Popup can be opened for the first time.    But when I narrowed the map,then maxzoom once again, popup unable to open   (My English is very poor, please forgive me)

**Comment by ghybs** [comment]
Hi,

Please can you provide more context and your code?
Which versions of Leaflet, Leaflet.markercluster and Leaflet.MarkerCluster.LayerSupport do you use?
What is your OS and browser, their versions?

Ideally reproduce your issue on [Leaflet playground](http://playground-leaflet.rhcloud.com/), [Plunker](http://plnkr.co/), or [JSFiddle](https://jsfiddle.net/), or at least share a link to your site where I can see the problem.

Otherwise, I am not able to help you if I cannot reproduce your issue.
The demo looks to be working fine for me: http://ghybs.github.io/Leaflet.MarkerCluster.LayerSupport/examples/mcgLayerSupport-controlLayers-realworld.388.html

**Comment by YoungHF** [comment]
Hi,

Thank you very much for your reply，demo of you offer is very good!
I'm very sorry I don't know how to share a link to my site yet...
The following is a part of the before I download the demo code,The other at the same code,This demo has a bug I mentioned above.(I hope you can understand.)

---

``` html
<head>

    <meta charset="utf-8" />
    <title>Leaflet.MarkerCluster.LayerSupport demo page (Layers Control)</title>

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.0-rc.1/leaflet.css" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.0-rc.1/leaflet.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="screen.css" />

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/1.0.0-rc.1.0/MarkerCluster.css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/1.0.0-rc.1.0/MarkerCluster.Default.css" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/1.0.0-rc.1.0/leaflet.markercluster.js"></script>

    <script src="../leaflet.markercluster.layersupport-src.js"></script>
    <script src="realworld.388.js"></script>

</head>
```

---

**Comment by ghybs** [comment]
Hi,

Thank you for the feedback.

Indeed I can reproduce the issue with Leaflet 1.0: http://plnkr.co/edit/fl1Idln4NvXkeLQ5ISbF?p=preview
(the repo demo still uses Leaflet 0.7)

Unfortunately there is no console error, so it may not be easy to troubleshoot.
In the meantime, if you are in a hurry, you can use Leaflet 0.7, or if you have a simple need, you might be interested in another plugin: [Leaflet.FeatureGroup.SubGroup](https://github.com/ghybs/Leaflet.FeatureGroup.SubGroup)

**Comment by YoungHF** [comment]
Hi,

OK.Thank you very much

**Comment by ghybs** [comment]
Should be fixed now in version 1.0.1
